### DISCUSSION TITLE
Fail2Ban on Restart

### DISCUSSION THREAD

**Discussion by ghost** [fail2ban/fail2ban#3199] [Q&A] [answered]
I am using fail2ban on a raspberry pie.  I noticed that when I restart, I have to re-start the service manually.  Is there a way I can start the service automatically?  I tried placing _systemctl start fail2ban_ in a cron job and putting an entry in my rc.local file and and neither worked successfully.

**Comment by sebres** [comment, marked as answer]
```bash
?sudo? systemctl enable fail2ban
```
(as any other systemd unit)

**Reply by ghost** [reply]
thanks.  turns out my telegram script was causing an error.
